main/test/ca/uwaterloo/flix/library/TestMutList.flix

```diff
@@ -1068,32 +1076,31 @@ namespace TestMutList {
     def reserve!01(): Bool & Impure =
         let v = MutList.new();
         MutList.reserve!(4, v);
-        MutList.capacity(v) == 4
+        capacity(v) - MutList.length(v) >= 4
```

Comment:
Is this test okay? It doe not test the capacity but rather the desired outcome of calling `reserve!`. Note that `capacity` is a helper function defined in the test suite.

Reply:
IMO yes, it's okay. I think it's normal that we should have to cheat a little bit in order to test a normally un-observable functionality.